src/redux/boards/actions.ts

```diff
@@ -134,8 +147,13 @@ export const actionCreators = {
     updateBackendBoard: (board: IBoard) => {
         return (dispatch: Dispatch) => {
             dispatch(actionCreators.updateBoardRequest(board))
-            return API.put('/boards', board).then(
-                response => dispatch(actionCreators.boardSuccess(response.message)),
+            return API.put(`/boards/${board.id}`, board).then(
+                response => dispatch(actionCreators.createBoardSuccess( response.board.id,
```

Comment:
Pas moyen que boardSucces prenne directement un board ?

Reply:
For createBoardSuccess, probably. 

I will apply the changes this afternoon.